src/Vatras/Translation/Lang/VariantList-to-VT.agda

```diff
@@ -84,15 +84,15 @@ fnoci-invariant : ∀ {ℓ} {A : Set ℓ} (x : A) (xs : List⁺ A) (n m i : ℕ)
   ≡ find-or-last (fnoci      n  (suc m) i c) (x ∷⁺ xs)
 fnoci-invariant x xs n m zero c z≤n = refl
 fnoci-invariant x xs n (suc m) (suc i) c (s≤s i≤m)
-  rewrite ∸-suc i≤m
+  rewrite +-∸-assoc 1 i≤m
```

Comment:
Interesting. I just opened a PR for adding `∸-suc` to the standard library but it already seems to exist. It was not obvious to me to look for `+-∸` interactions.